### ISSUE TITLE
script not uploading

### ISSUE THREAD

**Issue by sergeaubin** [bvandorp/formit2file#1]
i cannot get the script to work.  it does not upload and i do not get error.

**Comment by bvandorp** [comment]
Hi m8,

I retrieved the script back in the days from:
https://forums.modx.com/thread/47603/solved-file-upload-with-formit

I can see that there are more issues maybe yours is one of them?

**Comment by sergeaubin** [comment]
Hi Bart,

Thanks for replying.  I can't get passed    
if(move_uploaded_file($_FILES['ev_image']['tmp_name'], $target_path)) 
and im not sure why.
>>> Bart van Dorp <notifications@github.com> 6/27/2017 10:47 AM >>>
Hi m8,
I retrieved the script back in the days from:
https://forums.modx.com/thread/47603/solved-file-upload-with-formit
I can see that there are more issues maybe yours is one of them?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
(
https://github.com/bvandorp/formit2file/issues/1#issuecomment-311381846)
, or mute the thread
(

**Comment by sergeaubin** [comment]
Hey Bart,

I got it working using copy function instead of move_uploaded_file .

Question for you:  Have you any idea on how to fill in a map field in
my resource?  ie, street, housenumber, longitude, latitute

thanks

>>> Bart van Dorp <notifications@github.com> 6/27/2017 10:47 AM >>>
Hi m8,
I retrieved the script back in the days from:
https://forums.modx.com/thread/47603/solved-file-upload-with-formit
I can see that there are more issues maybe yours is one of them?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
(
https://github.com/bvandorp/formit2file/issues/1#issuecomment-311381846)
, or mute the thread
(
